### PR TITLE
Removed general landing markers for now pending pelican adjustment.

### PR DESCRIPTION
**Pull request by Flaksim**
Due to the Innies constantly rushing all the planets they can reach with their pelicans, and their pelicans currently having reach all over the overmap (which is unbalanced) I've removed their possible destinations save geminus.

No changelog, this is a backend change to stop the metarush.

### CODE REVIEW

File: maps/unsc_frigate/unsc_frigate-3.dmm [hunk 1]
```diff
@@ -215,877 +215,884 @@
...
-"kO" = (/obj/structure/cable/green{d1 = 1; d2 = 2; icon_state = "1-2"},/obj/effect/floor_decal/corner/blue{dir = 9},/turf/simulated/floor/tiled/white,/area/unscfrigate/medbay)
-"kP" = (/obj/effect/floor_decal/corner/blue{dir = 6},/turf/simulated/floor/tiled/white,/area/unscfrigate/medbay/pharmacy)
-"kQ" = (/obj/machinery/vending/medical,/turf/simulated/wall,/area/unscfrigate/medbay/pharmacy)
-"kR" = (/turf/simulated/wall,/area/unscfrigate/medbay/pharmacy)
-"kS" = (/obj/effect/wingrille_spawn/reinforced,/obj/machinery/door/firedoor,/turf/simulated/floor/tiled/white,/area/unscfrigate/medbay/pharmacy)
-"kT" = (/obj/machinery/door/airlock/halo{name = "Medbay Storage"; req_access = newlist()},/obj/machinery/door/firedoor,/turf/simulated/floor/tiled,/area/unscfrigate/medbay/pharmacy)
-"kU" = (/obj/machinery/door/airlock/halo_maint{name = "Maintenance"},/obj/machinery/door/firedoor,/turf/simulated/floor/plating,/area/unscfrigate/medbay)
-"kV" = (/turf/simulated/wall/r_wall,/area/unscfrigate/medbay)
-"kW" = (/obj/machinery/light{dir = 8},/ob
```
Comment (DoctorOctopusTheOctopi) [inline maps/unsc_frigate/unsc_frigate-3.dmm]: Unrelated line, just a tag for the file.. Remove all of the /obj/machinery/light/colored/red 's in the map. Travis hates it when those aren't used on GC for some reason. Someone should really globally define it.

**Review by DoctorOctopusTheOctopi — CHANGES_REQUESTED** [review]

File: maps/unsc_frigate/unsc_frigate-3.dmm (hunk 1 above)
Comment (caelaislinn) [reply maps/unsc_frigate/unsc_frigate-3.dmm]: An #include macro for the file containing the missing definition can be added in the includes file for the map. The red coloured light was originally a street object for the GC map though, not a warship warning light...

**Comment by caelaislinn** [comment]
Why close? EDIT: I see, you were splitting up the content into other PRs
